tests/utils/datatype_tests.py

```diff
@@ -104,3 +104,84 @@ def test_tile_clean(self):
         string.clean(tile2, nodeid)
 
         self.assertIsNotNone(tile2.data[nodeid])
+
+
+class ReferenceDataTypeTests(ArchesTestCase):
+    def test_validate(self):
+        reference = DataTypeFactory().get_instance("reference")
+
+        for value in [
+                "", 
+                [], 
+                [{}], # reference has no 'uri'
+                [{"uri":""}], # reference uri is empty
```

Comment:
The mockups I'm working from suggest that a URI is optional. We might need more direction here.

Reply:
URIs are required for references in business data (ie the URI is the thing being referenced), but should be optional for creating list items.  when they are not provided in the creation of list items we should generate a URI, but what that entails exactly still needs to be defined.